types/lhr/settings.d.ts

```diff
@@ -97,6 +97,8 @@ export type ScreenEmulationSettings = {
   precomputedLanternData?: PrecomputedLanternData | null;
   /** The budget.json object for LightWallet. */
   budgets?: Array<Budget> | null;
+  /** Run benchmarkindex this many extra times */
+  __internalMegaBenchmarkIndex?: number;
```

Comment:
see other comment: it's less weird to define this option as "run it this many times" vs "run it this many extra times"